backend/third_party_licenses/swf.csv

```diff
@@ -2,8 +2,7 @@
 github.com/kubeflow/pipelines, https://github.com/kubeflow/pipelines/blob/master/LICENSE, Apache-2.0
 github.com/kubeflow/pipelines, https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/archive/log.go, Apache-2.0
 cloud.google.com/go, https://github.com/googleapis/google-cloud-go/blob/v0.89.0/LICENSE, Apache-2.0
-cloud.google.com/go, https://github.com/googleapis/google-cloud-go/blob/v0.89.0/cmd/go-cloud-debug-agent/internal/debug/elf/elf.go, BSD-2-Clause
-cloud.google.com/go, https://github.com/googleapis/google-cloud-go/blob/v0.89.0/third_party/pkgsite/LICENSE, BSD-3-Clause
```

Comment:
@zijianjoy during version upgrade, two additional dependencies were added to cloud.google.com/go. Therefore, you'll need to add them to `subModules` section in go-licenses.yaml, to make sure we give credits to all the code that we use.

Reply:
refer to https://github.com/Bobgy/go-licenses/blob/fa4cc02071789620f9549a148b553ca3af4e188d/v2/config/config.go#L55-L66 for syntax. I need to add some user friendly documentation for the format.

Reply:
Thank you for the guidance! Noted about the usage of subModules. Because we are not going to release project cleaner and people importing KFP module rely on the root level go.mod, I am going to revert the changes to root and backend licenses and mods.